CONDA_RECIPE.md

```diff
@@ -0,0 +1,42 @@
+# Worflow to install the R package from github using conda
+
+1. Tag the repository
+```
+git tag -a v0.1
+git push origin --tags
+```
+
+2. Create conda recipe
+
+Suggestion: use a conda environment to build packages
+```
+conda create -n build-env conda-build
+conda activate build-env
+mamba install -c conda-forge r-base
+mamba install -c conda-forge r-essentials
+```
+
+Then,
+```
+conda skeleton cran git@github.com:PatZeis/scATAC.Peak.annotation.git
+```
```

Comment:
can you add here an alternative for people who have not set up an ssh key:
conda skeleton cran https://github.com/PatZeis/scATAC.Peak.annotation.git

Reply:
Yes, that's better. I changed it a bit: gave the esample using HTTPS and only mention that SSH can be used as an alternative.